source/docs/stats.md

```diff
@@ -136,7 +136,7 @@ should be created before the set is used for requests, via
 derived from data in a request, but there are limited set of known tokens, such
 as SSL ciphers or Redis commands.
 
-### Dynamic stat tokens
+### Dynnamic stat tokens
```

Comment:
typo

Reply:
Done; also added a diagram showing memory layouts of a few scenarios, and a table of all the classes related to symbol tables.